pyanaconda/modules/network/initialization.py

```diff
@@ -148,103 +148,6 @@ def _find_initramfs_connection_of_iface(self, iface):
         return None
 
 
-class ConsolidateInitramfsConnectionsTask(Task):
```

Comment:
I recommend to run [vulture](https://pypi.org/project/vulture/). There might be some functions and constants that are no longer used thanks to the cleanup.

Reply:
Good tip. I've found only https://github.com/rhinstaller/anaconda/pull/2894 which is unrelated to this patchset.